### ISSUE TITLE
MCS 3.11 PROD - Update netbackup to version 8

### ISSUE THREAD

**Issue by tbaker1313** [BCDevOps/developer-experience#646]
This will be tested in CHG0019300 on the PROD NFS Backup server OCIOPF-P-150.dmz on October 29th.
Depending on the results of this test, the update to the other nodes in MCS PROD 3.11 will be completed during OS patching November 21st - December 4th under CHG0018826

This will considered completed when all PROD 3.11 nodes that have backups are running the latest version of Netbackup.

**Comment by tbaker1313** [comment]
As per backup teams recommendation, new servers have been added to the configuration. Updates to NB8 on affected servers is still scheduled for OS PROD patching starting November 21st

**Comment by tbaker1313** [comment]
This is essentially completed. Outstanding steps are:
- update NB on MCS-Bastion
-  switch the order of servers in bp.conf (this requires the backup team to notify us first.)

**Comment by tbaker1313** [comment]
Still awaiting notification to switch the order of servers - otherwise this is completed.

**Comment by tbaker1313** [comment]
Still awaiting backup teams instructions to switch the server order.

**Comment by tbaker1313** [comment]
This is still awaiting one final step, waiting for backup team notification.

**Comment by tbaker1313** [comment]
Blocked as we are awaiting the backup team to tell us when we are ready to switch the servers.

**Comment by tbaker1313** [comment]
Completed on ociopf-d-100.dmz
Awaiting notifications from the backup team about the rest.

**Comment by wmhutchison** [comment]
Still waiting on DXCAS Backup team to inform us when remaining OCP3 servers need to be changed. Will be likely a race between completion of this versus a complete decomm of OCP3 in general.

**Comment by tbaker1313** [comment]
Adjusted a few more. There may be a few more servers who are still awaiting a change to their server order. Will close when we hear from the Backup team, or when the 3.11 clusters are decommissioned.

**Comment by tbaker1313** [comment]
Still in progress. This should be completed soon as I believe there's just a few hosts that have not been migrated.

**Comment by tbaker1313** [comment]
This is no longer needed as the cluster will be decommissioned.

**Comment by tbaker1313** [comment]
This is no longer needed as the cluster will be decommissioned.
